src/components/Filter.js

```diff
@@ -0,0 +1,24 @@
+import { useDispatch } from 'react-redux';
+import { filterCountry } from '../redux/filter/filter';
+
+const Filter = () => {
+  const dispatch = useDispatch();
+
+  const handleInput = (e) => {
+    const countryName = e.target.value.trim();
+    dispatch(filterCountry(countryName));
+  };
+
+  return (
+    <div className="filter">
+      <p>Stats by Country</p>
+      <input
+        id="submit"
+        placeholder="Search..."
```

Comment:
The Home page has a list of categories that could be filtered using given parameters, e.g. category name ✔